examples/resources/helmfeature/cl_resource.tf

```diff
@@ -0,0 +1,15 @@
+# Create Tanzu Mission Control cluster scope helm feature with attached set as default value.
+resource "tanzu-mission-control_helm_feature" "create_cl_helm_feature" {
+  scope {
+    cluster {
+      name            = "testcluster" # Required
```

Comment:
nit: There is terraform formatter target in Makefile , ensure to use it and fix the formatting for all files

Reply:
Updated PTAL